prov/verbs/src/verbs_eq.c

```diff
@@ -164,6 +165,9 @@ fi_ibv_eq_cm_process_event(struct fi_ibv_eq *eq, struct rdma_cm_event *cma_event
 	case RDMA_CM_EVENT_ESTABLISHED:
 		*event = FI_CONNECTED;
 		entry->info = NULL;
+		ret = fi_ibv_set_rnr_timer(cma_event->id->qp);
```

Comment:
rnr timer is an IB specific setting.  I'm not sure iwarp devices won't fail this call.